setup.py

```diff
@@ -2,5 +2,16 @@
 
 import setuptools
 
+
 if __name__ == "__main__":
-    setuptools.setup(use_scm_version=True)
+    setuptools.setup(
```

Comment:
Probably here it's best to revert to the left hand side.

Reply:
To be clear, do you want me to drop the "requires" and "tests_require" additions?

Reply:
Yes, that's right. I'm pretty sure that `requires` isn't even a supported keyword argument to `setuptools.setup()`. And `tests_require` should be in `setup.cfg` (under options.extras_require.testing).